x-pack/legacy/plugins/siem/public/components/ml/anomaly/use_anomalies_table_data.ts

```diff
@@ -40,31 +40,48 @@ export const getTimeZone = (config: Partial<AppKibanaFrameworkAdapter>): string
   }
 };
 
+export const getThreshold = (
+  config: Partial<AppKibanaFrameworkAdapter>,
+  threshold: number
+): number => {
+  if (threshold !== -1) {
+    return threshold;
+  } else if (config.anomalyScore == null) {
+    return 50;
+  } else if (config.anomalyScore < 0) {
+    return 0;
+  } else if (config.anomalyScore > 100) {
+    return 100;
+  } else {
+    return Math.ceil(config.anomalyScore);
```

Comment:
With the `Default anomaly threshold` setting description saying this value must be _exceeded_, does it make sense to `ceil` here as opposed to `floor`?

For instance, if the setting is `75.01` (as in your test above), it will be rounded up to `76`, and thus will only show scores `>76`, and not `76` itself, which is greater than the configured `75.01`. 

> <img src="https://user-images.githubusercontent.com/2946766/60834569-29c15c80-a17e-11e9-84f5-668736ceb485.png" width=400>

Reply:
I agree with you and adjusted that to be a `floor` rather than a `ceil` call.